Source/Scene/Cesium3DTileset.js

```diff
@@ -335,32 +355,32 @@ define([
          */
         this.tileVisible = new Event();
 
+        this._ready = false;
         this._readyPromise = when.defer();
 
         var that = this;
-
         this.loadTileset(tilesetUrl).then(function(data) {
             var tilesetJson = data.tilesetJson;
             that._asset = tilesetJson.asset;
             that._properties = tilesetJson.properties;
             that._geometricError = tilesetJson.geometricError;
             that._root = data.root;
-            that._readyPromise.resolve(that);
```

Comment:
@lilleyse any update here?  If resolving inside of the update is important, another solution would be to make sure we only reject inside the update as well, so it's at least consistent.  But I still think having to add it to the scene to have the promise resolve is odd (if not simply inconsistent with imagery)